.github/workflows/pr.yaml

```diff
@@ -61,6 +61,11 @@ jobs:
           - ":test:smoke:testApps:CoreAndFilter"
           - ":test:smoke:testApps:CoreAndFilter"
           - ":test:smoke:testApps:CoreAndFilter"
+          - ":test:smoke:testApps:CoreAndFilter"
```

Comment:
how does adding this same line fix the smoke tests?

Reply:
it's failing sporadically, so I'm running it lots of times to get a repro while I'm troubleshooting it

Reply:
oh, and I also updated azure sdk libraries, so maybe that fixed it?? I'll run a few 10s of more times to see...

Reply:
the sporadic failure is still there, I removed the debug code, so we can merge this PR since it's currently blocking #2214, and I'll continue troubleshooting the sporadic test seprately